MooseFS/moosefs-chunkserver/mfschunserver.cfg

```diff
@@ -0,0 +1,3 @@
+MASTER_HOST = $MOOSEFS_MASTER_SERVICE_HOST
+CSSERV_LISTEN_PORT = $MOOSEFS_CHUNKSERVER_SERVICE_PORT
```

Comment:
I am still not sure, how these environment variables are populated. Somewhere you will have to specify to the chunkservers how to connect to the master container. Could you please clarify how that is achieved?